examples/js/exporters/GLTFExporter.js

```diff
@@ -957,11 +956,35 @@ THREE.GLTFExporter.prototype = {
 
 				}
 
+				var interpolation;
+
+				// Detecting glTF cubic spline interpolant by checking factory method name because
+				// GLTFCubicSplineInterpolant is a custom interpolant and doesn't return
+				// valid value from .getInterpolation().
+				if ( track.createInterpolant.name === 'InterpolantFactoryMethodGLTFCubicSpline' ) {
```

Comment:
`.name` will disappear in some minified builds, I think... :/

Reply:
Ahhh, right. Hmmmmmm